react/EmptyStateMessage.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC, Fragment } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { useCssHandles } from 'vtex.css-handles'
+
+import EmptyStateIcon from './components/EmptyStateIcon'
+
+const CSS_HANDLES = ['emptyStateIconWrapper', 'emptyStateMessage'] as const
+
+const EmptyStateMessage: FC = () => {
```

Comment:
Is `EmptyStateMessage` a good name for this component? It has the icon too, right? And in the future, it can evolve to render more things since I feel that some stores might want to add some custom things to the empty state. What about naming it `EmptyState` or `EmptyCart`?

Reply:
I see what you mean! I went with this name since there is already an EmptyState component, which receives `children` and renders them.

To solve the problem regarding stores wanting to use more blocks inside this component, I think accepting `children` is the way to go.

Anyway, I've deleted these blocks in favor of creating this empty state using `flex-layout`, `icon-cart` and `rich-text` blocks :)